core-httpclient-impl/README.md

```diff
@@ -139,20 +151,49 @@ The polling interval is used to specify a fixed delay between consecutive HTTP r
 #### Initial datafile
 
 You can provide an initial datafile via the builder to bootstrap the `ProjectConfigManager` so that it can be used
-immediately without blocking execution.
+immediately without blocking execution. This also serves as a "fallback" datafile if no HTTP connection can be established.
+This can be useful in a mobile environment where internet connectivity is not always guaranteed. This datafile will be
+discarded, however, after the first successful datafile poll.
+
```

Comment:
Suggest revising to:

The initial datafile also serves as a fallback datafile if HTTP connection cannot be established. This is useful in mobile environments, where internet connectivity is not guaranteed. The initial datafile will be discarded after the first successful datafile poll.